types/recharts/index.d.ts

```diff
@@ -627,10 +628,10 @@ export interface SectorProps {
 }
 export class Sector extends React.Component<SectorProps, {}> {}
 
-export interface TextProps {
+export interface TextProps extends Partial<CSSStyleDeclaration> {
 	scaleToFit?: boolean;
 	angle?: number;
-	width?: number;
+	// width?: number;
```

Comment:
If the property is not there any more, could you please *remove* it instead of commenting it out?